tgstation.dme

```diff
@@ -6,6 +6,66 @@
 
 // BEGIN_FILE_DIR
 #define FILE_DIR .
+#define FILE_DIR "code"

```

Comment:
To get rid of these, in DM go to buuld > preferences and untick 'Automatically set FILE_DIR for subdirectories' then recommit the dme.